resources/public/api/conf/1.0/application.yaml

```diff
@@ -0,0 +1,67 @@
+openapi: "3.0.3"
+
+info:
+  version: "1.0"
+  title: Self Assessment Accounts (MTD)
+  description: |
+    List and retrieve details of Self Assessment balances, transactions, charges and payments.
+    
+    
+    # Send fraud prevention data
+    HMRC monitors transactions to help protect your customers' confidential data from criminals and fraudsters. 
+    <div class="govuk-warning-text warning-icon-fix">
+      <span class="govuk-warning-text__icon warning-icon-ui-fix" aria-hidden="true">!</span>
+      <strong class="govuk-warning-text__text">
+        <span class="govuk-warning-text__assistive">Warning</span>
+        You are required by law to submit header data for this API. This includes all associated APIs and endpoints.
+      </strong>
+    </div>
+    [Check the data you need to send](/guides/fraud-prevention/). You can also use the [Test API](/api-documentation/docs/api/service/txm-fph-validator-api/1.0) during initial development and as part of your quality assurance checks.
+    # Changelog
+    You can find the changelog in the [income-tax-mtd-changelog](https://github.com/hmrc/income-tax-mtd-changelog/#readme) GitHub wiki.
+    
+    
+    # Support
+    
+    * Raise non-technical or platform-related issues with the [Software Development Support Team (SDST)](https://developer.service.hmrc.gov.uk/developer/support).
+    * Raise technical issues on the [income-tax-mtd-changelog](https://github.com/hmrc/income-tax-mtd-changelog/issues) GitHub page.
+    * <a href="/contact/report-technical-problem?newTab=true&service=api-documentation-frontend" target="_blank">Is this page not working properly? (opens in new tab)</a>
+
+servers:
+  - url: https://test-api.service.hmrc.gov.uk
+    description: Sandbox
+  - url: https://api.service.hmrc.gov.uk
+    description: Production
+
+components:
+  securitySchemes:
+    User-Restricted:
+      type: oauth2
+      description: HMRC supports OAuth 2.0 for authenticating [User-restricted](https://developer.qa.tax.service.gov.uk/api-documentation/docs/authorisation/user-restricted-endpoints) API requests
+      flows:
+        authorizationCode:
+          authorizationUrl: https://api.service.hmrc.gov.uk/oauth/authorize
+          tokenUrl: https://api.service.hmrc.gov.uk/oauth/token
+          refreshUrl: https://api.service.hmrc.gov.uk/oauth/refresh
+          scopes:
+            write:self-assessment: Grant write access
+            read:self-assessment: Grant read access
+
```

Comment:
Could we add a tags section here? Something like [this](https://github.com/hmrc/property-business-api/blob/c8d4daf151e516da59371527d87f1cf645a0f4c5/resources/public/api/conf/1.0/application.yaml#L48)

Reply:
Added `Payments and Liabilities` & `Coding Out Underpayments and Debts` tags

Reply:
Thanks. Can we also then give each of the endpoints a tag so that they appear as a drop down under these two headings? Each endpoint will need a brief section like [this](https://github.com/hmrc/property-business-api/blob/6d636e91c0cabba66936c55be75b4d11e9923202/resources/public/api/conf/1.0/foreign_property_annual_submission_retrieve.yaml#L32)